storage/google/cloud/storage/constants.py

```diff
@@ -16,13 +16,28 @@
 # Storage classes
 
 STANDARD_STORAGE_CLASS = "STANDARD"
-"""Storage class for objects accessed more than once per month."""
+"""Storage class for objects accessed more than once per month.
+
+See: https://cloud.google.com/storage/docs/per-object-storage-class
```

Comment:
The link resolves to https://cloud.google.com/storage/docs/storage-classes , so let's just use that?

Also, I'm assuming you repeat this for every storage class so that it gets picked up in the generated docs, but is there a way to make it show up just once that would be intelligible? If not the duplication is fine.

Reply:
> The link resolves to https://cloud.google.com/storage/docs/storage-classes , so let's just use that?

03ba4372102

> Also, I'm assuming you repeat this for every storage class so that it gets picked up in the generated docs, but is there a way to make it show up just once that would be intelligible? If not the duplication is fine.

I don't see a good way do avoid repeating the URL without introducing lots of churn.

Reply:
Looks good! I will update/approve once the feature is no longer behind a flag (may be a couple of weeks).